frontend/src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -68,10 +72,11 @@ type ReduxDispatchProps = {
   toggleRank(): void;
   toggleServiceNodes(): void;
   toggleTrafficAnimation(): void;
+  toggleWaypoint(): void;
 };
 
-type GraphSettingsProps = ReduxStateProps &
-  ReduxDispatchProps &
+type ReduxProps = StateProps &
```

Comment:
I think this is not correct since `GraphToolbarState` type and `disabled` field are not part of Redux properties.

Reply:
Updated name